_posts/2023-03-09-group-equivariant-cnn-1/group-equivariant-cnn-1.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Upside-down, a cat's still a cat: Evolving image recognition with Geometric Deep Learning"
+description: >
+  In this first in a series of posts on group-equivariant convolutional neural networks (GCNNs), meet the main actors -- groups -- and concepts (equivariance). With GCNNs, after a break far too long, we finally revisit the topic of Geometric Deep Learning, a principled, math-driven approach to neural networks that has consistently been rising in scope and impact.
+author:
+  - name: Sigrid Keydana
+    affiliation: Posit
+    affiliation_url: https://www.posit.co/
+slug: keydanagcnn1
+date: 2023-03-09
+categories:
+  - Torch
+  - R
+  - Spatial Data
+  - Concepts
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/preview.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE, fig.width = 6, fig.height = 6)
+```
+
+This is the first in a series of posts on group-equivariant convolutional neural networks (GCNNs). Today, we keep it short, high-level, and conceptual; examples and implementations will follow. In looking at GCNNs, we are resuming a topic we first [wrote about](https://blogs.rstudio.com/ai/posts/2021-08-26-geometric-deep-learning/) in 2021: [Geometric Deep Learning](https://geometricdeeplearning.com/), a principled, math-driven approach to network design that, since then, has only risen in scope and impact.
+
+## From alchemy to science: Geometric Deep Learning in two minutes
+
+In a nutshell, Geometric Deep Learning is all about deriving network structure from two things: the domain, and the task. The posts will go into a lot of detail, but let me give a quick preview here:
+
+-   By domain, I'm referring to the underlying physical space, and the way it is represented in the input data. For example, images are usually coded as a two-dimensional grid, with values indicating pixel intensities. (At least, that's when the image has a single channel. Otherwise, there are a number of grids, each mapped to a separate channel.)
```

Comment:
Given it's just an example, I'd cut the parenthetical at the end here. If you really think the lack of nuance will confuse folks, maybe say "… single-channel images…"

Reply:
Yeah ... Totally makes sense (expecting some "Buuuuut ...", as always :-)))
I think a footnote would work well ...

Reply:
Yeah, a footnote will do. I think one of the hardest things in technical writing is to resist the urge to write for the "well, actually…" crowd. However, I think it's a habit worth fighting for the sake of most readers. 🙂